docs/readmes/proposals/p010_vendor_neutral_dp.md

```diff
@@ -416,7 +456,250 @@ scope of work.
 </tbody>
 </table>
 
-# Schedule and Milestones
+# Rationale
+
+A vendor neutral Domain Proxy provides a number of benefits to a CBRS
+deployment:
+
+-   The aggregation and proxy function allows for a single connection
+    between a customer network of CBSDs and the SAS. This provides a
+    small attack surface by reducing the number of outbound connections
+    traversing the customer firewall and edge devices.
+
+-   A domain proxy can be deployed in a controlled stable environment
+    allowing it to provide a consistent interface to the SAS on behalf
+    of a CBSD which may be subject to intermittent loss of power or
+    connectivity.
+
+-   It provides a point of coordination/control for radio channel
+    allocation in a multi-vendor deployment.
+
+-   It allows for the application of additional grant requesting and
+    maintenance logic which can correct errant behavior and/or provide
+    an optimization layer over vendor SAS client algorithms.
+
+
+# Compatibility
+
+Domain Proxy solution will reuse as much of the Magma services
+
+It should be easily deployed within existing Magma environment.
+
+As DP will be implementing radio intrefaces, the intention is to follow
+and comply with the Magma Community guidelines and codebase where applicable.
+
+# Observability and Debug
+
+The plan is to follow Magma best practises regarding application observability
+(logging, telemetry, debugging applications and the DP state)
+as well as utilize already existing tools and solutions in Magma infrastructure.
+
+# Implementation
+
+## System Architecture
+
+The following picture shows the DP components and the communication layout of the solution.
+
+<img src="media/p010_vendor_neutral_dp_image3.png" />
+
+---
+
+## DP Microservices
+
+* **Protocol Controller (PC)**
+  
+  Protocol Controller is responsible for receiving the initial traffic
```

Comment:
Basic question, is protocol controller and configuration/radio controller cohabited or are they running separately ? Curious to understand how CBSD is able to reach protocol controller ?

Reply:
CBSD will reach the protocol controller via an IP connection. CBSDs using the WInnForum protocol will have a configuration field for a URL of the SAS/DP. Similar for TR-069, protocol controller would be ACS (e.g. eNodebd), or SNMP server.